src/components/owasp_logging.py

```diff
@@ -0,0 +1,37 @@
+import logging
+
+from charmed_kubeflow_chisme.components import Component
+from ops import ActiveStatus, BoundStoredState, CharmBase, StatusBase
+from ops.charm import ConfigChangedEvent
+from owasp_logger import OWASPLogger
+
+logger = logging.getLogger(__name__)
+
+
+class OWASPLoggerComponent(Component):
+    def __init__(self, charm: CharmBase, stored: BoundStoredState, name="owasp-logger"):
+        super().__init__(charm, name)
+        self._charm = charm
+        self._owasp_logger = OWASPLogger(appid="minio.owasp-logger")
+        self._stored = stored
+        self._stored.set_default(last_secret_key_config="")
+
+    def _configure_app_leader(self, event):
+        if not isinstance(event, ConfigChangedEvent):
+            logger.info("Event is not a config-changed one. Skipping this component.")
```

Comment:
Do we need this log message?

Reply:
The charm would function without it 😄

I'd keep it as it can help us in the future always know why the function was called or not called, in case we are debugging.

Reply:
I think this one is still an `info`, can you update? :)